docs/patterns/lists.md

```diff
@@ -76,6 +76,8 @@ View example of the inline list pattern
 Apply the class `.p-inline-list--middot` to add a middot character between
 inline list items.
 
+As with any inline/inline-block element, any amount of white space (one or more spaces, tabs, etc.) in the markup is rendered as a single white space character. Vanilla uses the <a href="https://css-tricks.com/fighting-the-space-between-inline-block-elements/#article-header-id-3">font size method</a> to eliminate white space within a `p-inline-list--middot`, that is not inside a `p-inline-list__item`. White space within the `p-inline-list__item` however will displace the dot.
```

Comment:
You refer to CSS Tricks article by linking to "Set the font size to zero" section, while it seems what is actually use is [Negative margin](https://css-tricks.com/fighting-the-space-between-inline-block-elements/#article-header-id-1).

Also, a markdown notation to link could be used instead of HTML, unless there is some reason to use HTML here in the docs?
`[negative margin](https://css-tricks.com/fighting-the-space-between-inline-block-elements/#article-header-id-1)`.

Reply:
didn't know about the markup notation, I'll update, thanks

Reply:
@bartaz ok tried markdown, but how do you specify target="_blank"?